CHANGELOG.md

```diff
@@ -41,6 +41,17 @@ All notable changes to this project will be documented in this file.
 - `0.2.x` Releases - [0.2.0](#020)
 - `0.1.x` Releases - [0.1.0](#010)
 
+## [5.0.1](https://github.com/Nike-Inc/SQift/releases/tag/5.0.1)
+
+Release on 2019-06-13.  All issues associated with this milestone can be found using this
```

Comment:
`Release on 2019-06-13.  All` -> `Released on 2019-06-27. All`.

Reply:
Done

Reply:
Still not correct. Should be `Released` and there's an extra space between `. All`.